content/python/concepts/built-in-functions/terms/exec/exec.md

```diff
@@ -0,0 +1,100 @@
+--
+Title: 'exec()'
+Description: 'Executes a code object or string containing python code.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The **`exec()`** function executes a code object or string containing python code.
+
+## Syntax
+
+```pseudo
+exec(object, globals=None, locals=None)
+```
+
+* `object` : String or code object
+* `globals`: (optional) A dictionary containing global variables. If not specified, defaults to `None`. 
+* `locals`: (optional) A dictionary containing global variables. If not specified, defaults to `None`.
+
+The `exec()` function returns `None`.
+
+## Example 1
+
+This example uses `exec()` to parse and execute Python code contained in the string `code`:
+
+```py
+code = 'print("Hello, Codecademy!")'
+
+exec(code)
+```
```

Comment:
I'd like an output block after the example. It should look like this:

````
This example results in the following output:

```shell
Hello, Codecademy!
```
````